app/bundles/PageBundle/Model/TrackableModel.php

```diff
@@ -363,6 +363,7 @@ protected function extractTrackablesFromHtml($html)
         /** @var \DOMElement $link */
         foreach ($links as $link) {
             $url = $link->getAttribute('href');
+            $url = preg_replace('/{(\S+?)}/i', '', $url);
```

Comment:
What is the intent behind this replace? Do you expect whitespaces to be in the links? If so, shouldn't those be replaced with `+` or even better use https://www.php.net/manual/en/function.urlencode.php for the values in the GET params?